npc/other/CashShop_Functions.txt

```diff
@@ -310,3 +310,30 @@ function	script	F_Snowball	{
 	}
 	end;
 }
+
+// Status reduction potion
+//============================================================
+// - Permanently reduces base stat <type> by <val>.
+// - Returns status points equals to points needed to raise
+//   that stat to original value.
+// - Doesn't work if base status <type> would become lower than 1 after reduction.
+// * callfunc("F_CashReduceStat",<type>{,<val>,<itemid>});
```

Comment:
please correct the style issues in this function, spaces after `,` and `if` and parenthesis for script command calls.

Reply:
done